gce/teardown.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+
+MY_INSTANCE_NAME="my-app-instance"
+ZONE=us-central1-a
+
+gcloud compute instances delete $MY_INSTANCE_NAME \
```

Comment:
We should add region tags here and use them in the tutorial

Reply:
Nevermind, looks like this is not done in any other GCE tutorial, so this is good as-is!